src/displays/display-show-keys.html

```diff
@@ -0,0 +1,306 @@
+            </template>
+        
+        </div>
+
+    </template>
+    <script>
+
+        class DisplayLanguage extends new ReduxMixin(Polymer.mixinBehaviors([
+            Polymer.AppLocalizeBehavior,
+        ],
+            Polymer.Element
+        )) {
+            static get is() {
+                return 'display-show-keys';
+            }
+            static get properties() {
+                return {
+                    /**
+                    * Language is the users selected or determined language
+                    * @type {String}
+                    */
+                    language: {
+                        type: String,
+                        statePath: 'language',
+                    },
+
+                    /**
+                    * Path signlas to the parent to change the path
+                    * @type {Boolean}
+                    */
+                    path: {
+                        type: String,
+                        reflectToAttribute: true,
+                        notify: true,
+                    },
+                    locales: {
+                        type: Array,
+                    },
+                    /**
+                    * contains the public key
+                    * @type {String}
+                    */
+                    publicKey: {
+                        type: String,
+                        value: '0x339D7851463546164646546435468453546',
+                    },
+                    /*
+                    * toggles the view to show keys
+                    * @type {Boolean}
+                    */
+                    toggleKeys: {
+                        type: Boolean,
+                        value: true,
+                    },
+                    /*
+                    * toggles the view to show private key
+                    * @type {Boolean}
+                    */
+                    togglePK: {
+                        type: Boolean,
+                        value: false,
+                    },
+
+
+                };
+            }
+            /**
+            * Redux action for sertting language
+            * @param {language} language
+            */
+            static get actions() {
+                return Object.assign({
+                    LANGUAGE: function(language) {
+                        return {
+                            type: 'LANGUAGE',
+                            language: language,
+                        };
+                    },
+                });
+            }
+
+            connectedCallback() {
+                super.connectedCallback();
+                this._showButtons();
+                this.loadResources(this.resolveUrl('../text-translations.json'));
+            }
+
+            /**
+            * If we need to show buttons,
+            * get all the languages we have translations for
+            * @param {event} event represents the button the user clicked
+            */
+            _showButtons() {
```

Comment:
I dont understand why this is needed, as we have this on the language page

Reply:
Consider it gone.